sdk/python/feast/expediagroup/vectordb/milvus_online_store.py

```diff
@@ -58,9 +98,34 @@ def update(
         entities_to_keep: Sequence[Entity],
         partial: bool,
     ):
-        raise NotImplementedError(
-            "to be implemented in https://jira.expedia.biz/browse/EAPC-7970"
-        )
+        with MilvusConnectionManager(config.online_store):
+            for table_to_keep in tables_to_keep:
+                collection_available = utility.has_collection(table_to_keep.name)
+                try:
+                    if collection_available:
+                        logger.error(f"Collection {table_to_keep.name} already exists.")
```

Comment:
please make this log statement on "info" level. If a collection exists it is not an error and is expected.